src/common.h

```diff
@@ -5,16 +5,20 @@
  *      Author: root
  */
 
-#ifndef SRC_COMMON_H_
-#define SRC_COMMON_H_
+#pragma once
```

Comment:
Do we must use this?

Reply:
It's more elegant (saves the bottom #endif). Any reason against using this?

Reply:
Is it supported by all C compilers (or at least by the popular ones)?

Reply:
gcc, clang, msc and intel (same as gcc) support it.